src/docs/product/performance/index.mdx

```diff
@@ -26,6 +26,12 @@ Using the information on this page, you can trace issues back through services (
 
 ## Filter Performance Data
 
+<Alert level="info" title="Important">
+
+For organizations participating in the limited release, the transaction filter on the performance page has been simplified to support metric enhanced performance. The search bar can only be used to filter by transaction name. See [transaction search](/product/performance/filters-display/#transaction-search).
```

Comment:
```suggestion
For organizations participating in the limited release, the transaction filter on the Performance page has been simplified to support metric enhanced performance. The search bar can only be used to filter by transaction name. See [transaction search](/product/performance/filters-display/#transaction-search).
```

Also, I think Metrics Enhanced Performance is a term that we use internally only, so maybe we should say something else here instead?